### PR TITLE
Gutenberg: Remove VR Block

### PR DESCRIPTION
**Pull request by ockham**
Moving to the Calypso repo, see Automattic/wp-calypso#27873. See there for testing instructions.

Removing:
* The block JS and CSS
* The `gutenpack` target from the `gulpfile` (and renaming `gutenpack:jetpack-blocks` to `gutenberg:blocks`)
* The `.gitignore` line for blocks built via that target
* The `gulp-babel` dependency that was only used by that target (grep to verify!) 
* The script and style registers from `modules/shortcodes/vr.php` (we have functionality in `class.jetpack.php` for loading those)

Not removing the locale loading, since we continue to use that IIUC. See e.g. #10328 where they're modified.

Refer to #10328 for the things that were originally added when introducing the VR block.

### CODE REVIEW

**Comment by ockham** [comment]
Oh, I missed #9935...

**Comment by kraftbj** [comment]
Per https://github.com/Automattic/jetpack/pull/9935#issuecomment-430366057 , could you also remove the gulp script?

**Comment by ockham** [comment]
> Per https://github.com/Automattic/jetpack/pull/9935#issuecomment-430366057 , could you also remove the gulp script?

Yeah, I just wanted for #9042 to be merged to avoid a rebase. But I can do it already, that rebase shouldn't be too hard.

File: gulpfile.babel.js
Comment (tyxla) [inline gulpfile.babel.js]: Feels like this one could be named just `gutenpack` now, right?

Comment (sirreal) [reply gulpfile.babel.js]: Or something about blocks or Gutenberg assets.

**Comment by simison** [comment]
Original PR that added VR block added a bunch of other code to build the blocks as well, suppose we don't need those anymore neither?

https://github.com/Automattic/jetpack/pull/9560

File: gulpfile.babel.js
Comment (kraftbj) [reply gulpfile.babel.js]: Y'all good with `gutenpack:jetpack-blocks`?

**Comment by kraftbj** [comment]
cc: @gravityrail the VR shortcode added some things and it's time to remove it to defer to the One Repo. Wanted to loop you in on if any of the other things from 9560 needed to go or needed to stay specifically.

File: gulpfile.babel.js
Comment (ockham) [reply gulpfile.babel.js]: (`jetpack` is kinda implicit anyway, since this is the Jetpack codebase :smile: )

**Comment by ockham** [comment]
Should be ready for review now.

**Review by kraftbj — DISMISSED** [review]
Built fine, everything looks good. Please feel free to merge at will once Luna has reviewed.

**Comment by simison** [comment]
https://github.com/Automattic/jetpack/pull/9560 added i18n stuff as well:

https://github.com/Automattic/jetpack/blob/9c660da25b85de48fed6e85afde7d0145b530124/class.jetpack.php#L3846-L3851

— do you recon we want to keep that around because we might need it soon, or perhaps something else than the VR block is using it as well?

Otherwise we should probably clean it out not to leave un-used code around.

**Comment by simison** [comment]
@sirreal answered my question elsewhere: https://github.com/Automattic/jetpack/pull/10328#discussion_r226581432

>Additionally, I expect the view scripts to depend on wp-i18n in the near future (#10319).
>
>Let's leave it as it.

**Comment by ockham** [comment]
Thanks @simison! I'd actually added a note to the PR desc :grin: 

> Not removing the locale loading, since we continue to use that IIUC. See e.g. #10328 where they're modified.

Had to rebase, so I need another approval pls :smile:

**Review by tyxla — APPROVED** [review]
This is building well and looking good to me 👍

**Review by jeherve — APPROVED** [review]
LGTM! 👍

**Pull request merged**
